sdk/communication/Azure.Communication.Chat/src/Models/CommunicationUserIdentifierModel.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.Communication
 {
     [CodeGenModel("CommunicationUserIdentifierModel")]
-    internal partial class CommunicationUserIdentifierModel
+    public partial class CommunicationUserIdentifierModel
```

Comment:
`internal`

Reply:
set to public since CommunicationIdentifierModel needs to be public (reasoning as above)

Reply:
replaced by internal